src/main/java/core/basesyntax/Colores.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public enum Colores {
```

Comment:
```suggestion
public enum Color {
```

enum it is class and we should use singular for class names